src/lib/structures/Command.ts

```diff
@@ -108,7 +107,5 @@ export interface CommandOptions extends AliasPieceOptions {
 	preconditions?: PreconditionContainerResolvable;
 	extendedHelp?: string;
 	flags?: string[];
-	guarded?: boolean;
-	hidden?: boolean;
-	quotedStringSupport?: boolean;
+	quotes?: [string, string][];
```

Comment:
Optional
```suggestion
	quotes?: [openingQuote: string, closingQutoe: string][];
```

Reply:
![image](https://user-images.githubusercontent.com/24852502/91104216-9c84e400-e66d-11ea-98c4-3611f0ea8b49.png)

Reply:
Update your TS boi

Reply:
![image](https://user-images.githubusercontent.com/24852502/91104398-fdacb780-e66d-11ea-92ad-c2584ebb446a.png)

Reply:
🤷‍♂️ prettier needs to update itself